controller.py

```diff
@@ -63,6 +63,7 @@
 from core.util.http import HTTP
 from core.util.opds_writer import OPDSMessage
 from core.util.problem_detail import ProblemDetail
+from core.util.string_helpers import base64
```

Comment:
We use `base64` in a part of the file that isn't mentioned in this diff. The part that shows up in the diff is where we still need to use stdlib's base64, which has been renamed.

Reply:
But is this until the conversion or will this remain?

Reply:
I think we'll keep this, because it makes it easy to base64-encode a Unicode string in Python 3 and get a Unicode string back.